packages/editor/src/components/post-publish-panel/style.scss

```diff
@@ -188,3 +188,46 @@
 		}
 	}
 }
+
+.edit-post-layout,
```

Comment:
What are these styles about? Can we have them here without wrappers?

Reply:
They are mostly for repositioning the publish panel in those editors and I'm not sure if we should have them without wrappers, because the panel is public API..

Reply:
I think we should probably leave these in their own packages until we move the "interface skeleton" unless all the classes that we target are within the "editor" package.

Reply:
Yes, the targeted classes are within editor package.